politeiawww/comments.go

```diff
@@ -21,6 +20,17 @@ import (
 	"github.com/decred/politeia/util"
 )
 
+const (
```

Comment:
This needs to be in the API. I get it has to be an int but there can be a named constant `VoteActionDown` and `VoteActionUp`. These should be right above the struct and documented what they do. The struct documentation where it is used should point to the constants.

Sorry that I did not make this more obvious when I requested it.

Reply:
> I get it has to be an int

*string

The API has them defined as strings.

Reply:
@marcopeereboom sorry, I am a bit lost .. which `API` and which `struct` are you referring to?

Reply:
Made the changes requested. PTAL. Thanks.